package.json

```diff
@@ -26,7 +26,7 @@
     "lint:js": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore \"**/*.{cjs,js,md,mjs}\"",
     "lint:prettier": "prettier --cache --cache-location .cache/prettier --cache-strategy content --check \"**/*.{cjs,js,json,md,mjs,scss,yaml,yml}\"",
     "lint:scss": "stylelint \"**/*.{md,scss}\"",
-    "check-links": "hyperlink --canonicalroot https://design-system.service.gov.uk/ --internal --recursive deploy/public/index.html | tap-mocha-reporter min"
+    "check-links": "hyperlink --canonicalroot https://design-system.service.gov.uk/ --internal --root deploy/public $(glob \"deploy/public/**/*.html\") | tap-mocha-reporter min"
```

Comment:
So was `--recursive` doing something funky where it kinda radiated out of index.html? And now we're just looking at each file?

Reply:
Yeah when the [`--internal` flag was removed](https://github.com/alphagov/govuk-design-system/pull/3174) it started flagging at _external_ website issues etc 😬

But in this PR since we're keeping `--internal` AND the [linked comment](https://github.com/Munter/hyperlink/issues/179#issuecomment-619537282) seems to indicate the problem lies with `hyperlink` streaming to the TAP reporter we could try switching `glob` back to `--recursive`?

I just tried everything possible until it went stable

Reply:
Worth a go? Makes the script a little easier to parse.

Reply:
Hyperlink also recommends using a sitemap, so if the problem is the reporter, we could probably reinstate that too. Not as clear how beneficial `--source-maps` is.

Reply:
Yeah worth a try, pushed it up so we can do another 10 runs 😆